### PR TITLE
docs(PPDSC-2800) inline-message-comp-update

### PR DESCRIPTION
**Pull request by RashikaNewsUK**
PPDSC-2800

**What**

1. Background - why this is needed
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [x] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [x] The screen reader reads and flows through the elements as expected.
- [x] There are no new errors in the browser console coming from this PR.
- [x] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2800-update-inline-message-component
  Storybook: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2800-update-inline-message-component/storybook

File: src/inline-message/__tests__/inline-message.stories.tsx [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
-          borderWidth: '1px 4px',
-          borderColor: 'deeppink',
+          backgroundColor: '#FDDCC6',
+          borderWidth: '0px 0px 0px 4px',
+          borderColor: '#804200',
           borderStyle: 'solid',
-          borderRadius: '3px',
-          iconColor: 'deeppink',
+          borderRadius: '4px',
+          iconColor: '#804200',
```
Comment (mutebg) [inline src/inline-message/__tests__/inline-message.stories.tsx:28]: Do we have tokens for these?

Comment (RashikaNewsUK) [reply src/inline-message/__tests__/inline-message.stories.tsx:28]: @mutebg there is no tokens for this!

Comment (RashikaNewsUK) [reply src/inline-message/__tests__/inline-message.stories.tsx:28]: @mutebg  Shall I create for it?

**Review by mutebg — APPROVED** [review]

**Pull request merged**